### DISCUSSION TITLE
ADR 0. Record architecture decisions in the repository.

### DISCUSSION THREAD

**Discussion by beechnut** [18F/NRM-Grants-Agreements#362] [Architectural Decision Records]
View the proposal: [ADR 0. Record architecture decisions in the repository.][link]

__Summary__
> In order to support clear decision-making, enable a smooth vendor transition, arrive at appopriately stable technical decisions, and provide a clear historical record of technical decisions, we are proposing the use of Architectural Decision Records (ADRs).

[link]: https://github.com/USDAForestService/NRM-Grants-Agreements/blob/add-architecture-decision-records/decisions/architecture/000-record-architecture-decisions-in-the-repository.md

**Comment by neilmb** [comment]
I strongly support this. I suspect we'll need to do some coaching to help the three responsible parties to decide and record the decision.
